torch/ao/quantization/experimental/apot_utils.py

```diff
@@ -48,3 +48,11 @@ def float_to_reduced_precision(x, levels, indices):
 def apot_to_float(x_apot, levels, indices):
     idx = list(indices).index(x_apot)
     return levels[idx]
+
+r"""Clips value x based on alpha qparam
+"""
+def clip(x, alpha):
```

Comment:
can just use torch.clamp

Reply:
also, you should either be clipping positive and negative, or not be clipping at all.